backend/app/services/universal_loader/models.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
+from pydantic import BaseModel
+
+
+class OpenAI_Dataset_Format(BaseModel):
+    content: Optional[str]
+    role: Optional[str]
+    createdAt: Optional[str]
+    conversationId: Optional[str]
+
+
+class Phospho_Dataset_Format(BaseModel):
+    input: Optional[str]
+    output: Optional[str]
+    created_at: Optional[str]
+    task_id: Optional[str]
+    session_id: Optional[str]
+
+
+class User_assistant(BaseModel):
```

Comment:
Please rename this user PascalCase (no underscore and every word starts with a capital)